elements/pfe-band/src/pfe-band.scss

```diff
@@ -103,246 +179,206 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
-            "aside header"
-            "aside body"
-            "aside footer";
-        }
+      :host([has_aside][aside-desktop="left"]) & {
+        grid-template-rows: auto;
+        grid-template-columns: pfe-local(layout, $fallback: pfe-local(Width__aside--lg) 1fr);
       }
     }
   }
+}
 
-  // For each section in the band layout
-  @each $section in (header, body, aside, footer) {
-    &__#{$section} { // i.e., pfe-band__header
-      margin-bottom: pfe-local(gutter--vertical);
-      // Apply the standard grid
-      @supports (display: grid) {
+.pfe-band__container,
+.pfe-band__wrapper {
+    // Fallback margins if grid is not used
+    margin-bottom: calc(-1 * #{pfe-local(gutter--vertical)});
+    margin-right: calc(-1 * #{pfe-local(gutter--vertical)});
+    @supports (display: grid) {
+      margin: 0 auto;
+    }
+}
+
+.pfe-band__aside {
+  flex-grow: 1;
+  align-self: flex-start;
+}
+
+@each $section in (header, body, footer, aside) {
+  .pfe-band__#{$section} {
+    // Fallback margins if grid is not used
+    margin-bottom: pfe-local(gutter--vertical, $region: $section);
+    margin-right: pfe-local(gutter--vertical, $region: $section);
+
+    // Apply the grid layout
+    @supports (display: grid) {
+      // Apply the section as a named grid-area
+      grid-area: $section;
+      margin-bottom: 0;
+      margin-right: 0;
+
+      :host([use-grid]) & {
         display: grid;
-        row-gap: pfe-local(gutter--vertical);
-        column-gap: pfe-local(gutter--horizontal);
-        margin-bottom: 0;
-        // Apply the section as a named grid-area
-        grid-area: $section;
-        // Allow users to hook into the layout with custom columns
-        grid-template-columns: pfe-local(layout, $region: #{$section});
-        grid-template-rows:    auto;
+        grid-template-rows: auto;
+        grid-template-columns: pfe-local(layout, $region: $section);
+
+        grid-row-gap: pfe-local(gutter--vertical, $region: $section);
+        row-gap:      pfe-local(gutter--vertical, $region: $section);
+        grid-column-gap:  pfe-local(gutter--horizontal, $region: $section);
+        column-gap:       pfe-local(gutter--horizontal, $region: $section);
       }
     }
   }
+}
 
-  &__aside {
-    align-self: flex-start;
+// For each section in the band layout,
+// define their internal behavior for slotted elements
+@include browser-query(ie11) {
+  .pfe-band__container {
+    @include pfe-clearfix;
   }
 
-  @include browser-query(ie11) {
-    &__body {
-        width: 60%;
-        float: left;
+  .pfe-band__header,
+  .pfe-band__body,
+  .pfe-band__footer {
+    float: left;
+
+    :host([aside-mobile="top"]) & {
+      float: right;
+    }
+  
+    :host(:not([aside-height="full"])) & {
+      width: 60%;
     }
-    &__aside {
+  }
+
+  .pfe-band__footer {
+    clear: both;
+  }
+
+  .pfe-band__aside {
+    flex-grow: 1;
+    align-self: flex-start;
+
+    float: right;
+    
+    width: 35%;
+    margin: 0 2.5%;
```

Comment:
This margin `0 2.5%` may be what's causing the aside region to drop down in ie11 (rounding errors?) If I change it to `0 1.9%` it jumps back up.

Reply:
Pushed that up